### PR TITLE
[Devops-2968] replace human creds with api token

### PR DESCRIPTION
**Pull request by sonuGlobality**
Replaces user/pass with pypi token that will prevent build failures on passwords rotation

### CODE REVIEW

File: entrypoint.sh [hunk 1]
```diff
@@ -28,7 +28,8 @@ elif [ "$1" = "lint" ]; then
    # Install standard linting dependencies; YMMV
    pip --quiet install \
        .[lint] flake8 flake8-print flake8-logging-format flake8-isort
-   exec flake8 ${NAME}
+   flake8 ${NAME}
+    exit
```
Comment (simpleigh-globality) [inline entrypoint.sh:32]: What does this do? Also seems to be misaligned?

Comment (simpleigh-globality) [reply entrypoint.sh:32]: Not sure it's needed TBH as we're in an `if` statement and the script terminates afterwards.

Comment (simpleigh-globality) [reply entrypoint.sh:32]:
```suggestion
```

**Review by tonybajan-globality — APPROVED** [review]
lgtm once leigh's note is addressed

File: entrypoint.sh (hunk 1 above)
Comment (sonuGlobality) [reply entrypoint.sh:32]: I don't see any need for exit here but la me ask @marcglobality 

@marcglobality 
This change was introduced by you in this PR : https://github.com/globality-corp/globality-build/pull/480/files
Any comment?

Comment (simpleigh-globality) [reply entrypoint.sh:32]: Created https://github.com/globality-corp/globality-build/pull/483

Comment (sonuGlobality) [reply entrypoint.sh]: merged

**Pull request merged**